cardlibrary/ancientstirrings.lua

```diff
@@ -477,9 +477,9 @@ local stirrings = { -- CARD_ID, NAME, POWER, HEALTH, RARITY,BIO
 		["Cost"] = {["Green"] = 3,},
 		["Effect"] = {
 			Name = "Pyrotechnics",
-			Description = "When targeted, transform into an Overseer Beast.",
+			Description = "When targeted, give all fighters 150 health and power, all Overseer fighters 150 health and power, and transform it into an Overseer Eye.",
```

Comment:
Rewrite the effect desc. as "When targeted, give all fighters 150 Health and Power, give all Overseers an additional 150 Health and Power, and transform this card into an Eye of Overseer.